v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/AvroTypeConvertorException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.transformer;
```

Comment:
This is the right directory to write this code? 
Shouldn't this code be localised to some Spanner specific directory? This looks off, makes it seem like `tranaformer` is a teleport wide construct?

If this is indeed the right directory, please include this directory in CodeCov config and exclude the exception classes specifically from coverage metrics.

@darshan-sj Please help.

Reply:
The directory is inside sourcedb-to-spanner, which is the template specific directory. All template package names start with `com.google.cloud.teleport.v2` despite being inside their own template directories.

The directory seems to be already included as I a few codeCov warnings. Will remove the exception classes.

Reply:
Understood, thank you!

Reply:
Actually, these should be in `spanner-common` right? As per the other comment this is the generic converter class we will use.

Reply:
Correct, I'll move it.